views/package_suppliers.dt

```diff
@@ -0,0 +1,37 @@
+extends layout
+
+block title
+	- auto title = "Package suppliers";
+
+block body
+	h2 Introduction
+
+	p Dub supports package suppliers of type Dub Registry, Maven and File System. By default Dub will search on Dub Registry <a href="https://code.dlang.org">https://code.dlang.org</a> and various fallbacks.
+
+	p Package suppliers can be specified either using argument <code>--registry</code> or using setting <code>registryUrls</code>. The type of the package supplier could be specified using a prefix <code>dub+</code>, <code>mvn+</code> or <code>file://</code>. By default type Dub Registry is assumed.
+
+	h3 Dub Registry
+
+	p Dub packages are registerd at a Dub Registry server like <a href="https://code.dlang.org">https://code.dlang.org</a>. It isn't needed to prefix the address of a Dub Registry.
+
+		ul
+			li https://code.dlang.org/
+			li dub+https://code.dlang.org/
+
```

Comment:
Maybe add a paragraph linking to https://github.com/dlang/dub-registry here

Reply:
fixed